### DISCUSSION TITLE
shift+enter [May 06]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2040] [Discord Threads]
> 🙋‍♂️ **TrevLev**

**Comment by o1lab** [comment]
> 💁 **o1lab**  
#### we can take that as enhancement and add it in next release  
#  
> 🙋‍♂️ **TrevLev**  
#### Great, thanks!  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/969621275511250994)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
